src/components/Tag/Tag.tsx

```diff
@@ -10,40 +10,73 @@ import styles from './Tag.scss';
 
 export interface NonMutuallyExclusiveProps {
   /** Content to display in the tag */
-  children?: string;
+  children?: string | React.ReactElement;
```

Comment:
```suggestion
  children?: React.ReactNode;
```

`ReactNode` is the type we generally go for to allow custom markup in Polaris components. And string will also be included in `ReactNode`!

String is included in React

Reply:
Sweet!!!